### ISSUE TITLE
Translate problem Nightly

### ISSUE THREAD

**Issue by Schn1ek3** [theheroGAC/Autoplugin#157]
Autoplugin Nightly hangs when I want to translate SD2VITA_INVALID_CONFIG

**Comment by bamhm182** [comment]
You're using Chinese Traditional, correct?

**Comment by Schn1ek3** [comment]
I translate in German. 
It hangs but doesn't crash. 

Thank you :)

**Comment by bamhm182** [comment]
The problem is that "\n" is breaking the on screen keyboard. osk.init("TITLE", "TEXT") will bring up the on screen keyboard with the title "TITLE" and the word "TEXT" pre-filled. If I put osk.init("TITLE", "TEXT\nTEXT") it crashes, if I put osk.init("TITLE\nTITLE", "TEXT") it does not... I'll have to work on this later, but I think that a solution may be to just not pre-fill the text field with anything if it has "\n"... This will be a little annoying, but it is better than it crashing.

**Comment by theheroGAC** [comment]
yes is available https://github.com/theheroGAC/Autoplugin/blob/master/Nightly/RELEASE/Autoplugin%20Nightly.vpk

**Comment by Schn1ek3** [comment]
Yes now it works :) Thank you :)
